### ISSUE TITLE
Funicular vs. dam

### ISSUE THREAD

**Issue by LucFreitas** [gravitystorm/openstreetmap-carto#3700]
The representation is very similar in both cases.

waterway=dam:
![1](https://user-images.githubusercontent.com/19976610/53592207-8bd8d800-3b74-11e9-92ff-c95c3e451f5c.jpg)

railway=funicular:
![2](https://user-images.githubusercontent.com/19976610/53592208-8bd8d800-3b74-11e9-9a91-ada0040833e6.jpg)

Some hydroelectric dams contain funicular systems. Does this create any confusion?

**Comment by matkoniecz** [comment]
> Does this create any confusion

Yes, unfortunately with some displayed features it is hard to avoid conflicts.

But I would expect that in general confusion should be unlikely thanks to a context. Are you sure that examples are correctly tagged?

**Comment by StyXman** [comment]
Maybe make dams thicker?

**Comment by jeisenbe** [comment]
@LucFreitas - could you provide a link to the locations or the OSM elements shown above?

**Comment by StyXman** [comment]
Both are here, but the dam seems to have been removed: https://www.openstreetmap.org/#map=17/-20.08644/-40.55763
